### DISCUSSION TITLE
How can I insert a line break before a specific character?

### DISCUSSION THREAD

**Discussion by nicewillsuccess** [ONLYOFFICE/DocumentBuilder#142] [Q&A] [answered]
I want to insert a line break before a specific character. I considered using '\n' before the character, but it affects the style and is quite complex to handle. I noticed that in the documentation for text editors, only the run object can add line breaks, but it cannot be inserted before a specific character. Is there a better solution?
![image](https://github.com/user-attachments/assets/b558a0a1-20e0-499c-89e7-4aba8a4ae356)

**Comment by trofim24** [comment, marked as answer]
You can insert a line break using `AddLineBreak` [method](https://api.onlyoffice.com/docs/office-api/usage-api/text-document-api/ApiRun/Methods/AddLineBreak/)
